docs/README.md

```diff
@@ -1,3 +1,15 @@
+Personal note:
+I thought this weeks challenge went reasonably well. I started off with the 'peep' class, added the method .all, .create and as an extra, .delete. I then created a User class, which asks users for their name, email and password via the /signup route.
+
+I then wanted to add some optional styling so for this I used bootstrap, combined with some extra CSS styling which can be found i the layout.erb file.
+
+To set up the databases please refer to docs/01. set up databases.
```

Comment:
perhaps you could add a section covering this in a little more detail?

How to setup the development database from scratch:

1. Instal psql
2 Start psql
3. Create the database using the psql command CREATE DATABASE "<dbname>";
4. Connect to the database using these commands:
psql
\c <dbname>;
5. Run the queries saved in docs.